encoding/codecv0.go

```diff
@@ -0,0 +1,409 @@
+package encoding
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"math"
+	"reflect"
+
+	"github.com/scroll-tech/go-ethereum/common"
+	"github.com/scroll-tech/go-ethereum/core/types"
+	"github.com/scroll-tech/go-ethereum/crypto"
+	"github.com/scroll-tech/go-ethereum/crypto/kzg4844"
+)
+
+type DACodecV0 struct{}
+
+// Version returns the codec version.
+func (o *DACodecV0) Version() CodecVersion {
```

Comment:
Why `o` as the receiver?

Reply:
just personal preference, may "d" be more suitable in Go idiomatic namings? wdyt?

Reply:
changed from "o" to "d" in [986850d](https://github.com/scroll-tech/da-codec/pull/25/commits/986850d727f596b38a8accaf5f313d2f41e8e601).